src/clarity-angular/datagrid/datagrid-filter.spec.ts

```diff
@@ -36,7 +36,8 @@ export default function(): void {
 
             it("registers to the FiltersProvider provider", function() {
                 expect(filterService.getActiveFilters()).toEqual([]);
-                component.setFilter( filter );
+                // component.setFilter( filter );
+                component.customFilter = filter;
```

Comment:
You have a few more of these in this `datagrid-filter.spec.ts` file, that's what I meant by replacing all occurrences in this spec. I can't comment on them because they haven't change, but I see them on lines 45 and 53.